unstructured_ingest/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.26-dev3"  # pragma: no cover
+__version__ = "0.0.27"  # pragma: no cover
```

Comment:
Always include a `-dev` in the changelog update, I'll coordinate to bump the version for a release.